Makefile

```diff
@@ -27,7 +27,15 @@ protobufs: $(PY)
 %_pb2.py: %.proto
 	protoc --python_out=. --mypy_out=. $<
 
-.PHONY: clean
-
-clean:
+clean-proto:
 	rm -f $(PY) $(PYI)
+
+clean-build:
```

Comment:
I would prefer to just call this `clean` but since there was already a `clean` target, I didn't want to leave any ambiguity for people who were used to the old target.

Still, I think that we want `clean` to be something that we're comfortable doing before every release, and running the old `clean` definitely causes problems without doing some kind of follow-up commands (which I didn't have a chance to look into).

Reply:
Yeah, agree. Old `clean` actually removes the compiled `pb2.py` and the mypy stubs from the pb(protobuf) files. I'm for having `clean-build` as `clean` actually, since `rm -f $(PY) $(PYI)` is not frequently used from my experience.

Reply:
Sold! I made the change.